dockstore-webservice/src/main/resources/migrations.1.12.0.xml

```diff
@@ -387,4 +387,10 @@
             alter table sourcefile enable row level security;
         </sql>
     </changeSet>
+    <changeSet author="dyuen (generated)" id="addAppToolEvents">
+        <addColumn tableName="event">
+            <column name="apptoolid" type="int8"/>
```

Comment:
Out of curiosity, is there ever a practical difference between `int8` and `bigint`?  (I don't know the answer.)

Reply:
Nope, looks like its just an alias https://www.postgresql.org/docs/12/datatype-numeric.html